lighthouse-core/audits/byte-efficiency/unminified-javascript.js

```diff
@@ -83,7 +83,12 @@ class UnminifiedJavaScript extends ByteEfficiencyAudit {
       if (!script.content) continue;
 
       const networkRecord = getRequestForScript(networkRecords, script);
-      const displayUrl = script.name === artifacts.URL.finalUrl ?
+
+      // The script URL will match the main document URL if it is inline.
+      // It will not necessarily match the frame URL.
+      // For timespan mode, this means we can't use this fancy display URL for inline scripts.
+      // TODO: Determine inline scripts in timespan mode.
+      const displayUrl = script.url === artifacts.URL.mainDocumentUrl ?
```

Comment:
copy/paste from a branch I have not made a PR for yet:

```
+      // If this is an inline script, place the node inside a top-level (aka depth-one) node.
+      // Also separate each iframe / the main page's inline scripts into their own top-level nodes.
+      const isInlineHtmlScript = script.startColumn || script.startLine;
```

Reply:
startColumn is most certainly non-zero for all inline scripts, as the position the scripts starts is only after `<script>` which is 8 characters.